packages/environment/package.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "@sveltejs/environment",
+	"version": "1.0.0-next.1",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/sveltejs/kit",
+		"directory": "packages/environment"
+	},
+	"license": "MIT",
+	"homepage": "https://kit.svelte.dev",
+	"type": "module",
+	"exports": {
+		".": {
+			"browser": {
+				"development": "./dev-browser.js",
+				"default": "./prod-browser.js"
+			},
+			"default": {
+				"development": "./dev-ssr.js",
+				"default": "./prod-ssr.js"
+			}
+		},
+		"./package.json": "./package.json"
```

Comment:
do we need the package.json export? in a discussion in vite-plugin-svelte repo it was brought up that this isn't strictly needed.

Reply:
`rollup-plugin-svelte` still requires it at the moment, but we should work towards removing that. I figured it's good to keep it here until then as it doesn't hurt anything to have it